docs/website/src/site/styles/homepage/explore.scss

```diff
@@ -0,0 +1,205 @@
+@import './../breakpoints';
+
+$transition-timing: 300ms;
+
+.ds-explore {
+  margin-top: 3.2rem;
+
+  @media screen and (min-width: $md-breakpoint) {
+    margin-top: 4rem;
+  }
+
+  @media screen and (min-width: $xl-breakpoint) {
+    margin-top: 4.8rem;
+  }
+
+  h2 {
+    font-size: 2.4rem;
```

Comment:
why not use variables for this? so you can make sure the headings are the same all over the pages?

Reply:
in other places h1 has 2.4rem (in general), but we already have h1 on the homepage, maybe I should use in this example `ds-heading-2` class instead. :)